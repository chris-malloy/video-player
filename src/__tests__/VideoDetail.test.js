import React from 'react';
import VideoDetail from '../components/VideoDetail';
import renderer from 'react-test-renderer';

describe('Component: VideoDetail', () => {

    test('VideoDetail renders correctly', () => {
        const component = renderer.create(
            <VideoDetail>VideoDetail</VideoDetail>
        )
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});