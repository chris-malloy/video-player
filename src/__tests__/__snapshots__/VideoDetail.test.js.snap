// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: VideoDetail VideoDetail renders correctly 1`] = `
<div>
  Loading...
</div>
`;
